app/scripts/filters/resources.js

```diff
@@ -238,12 +97,6 @@ angular.module('openshiftConsole')
       return icon;
     };
   })
-  .filter('imageStreamTagIconClass', function(imageStreamTagAnnotationFilter) {
-    return function(resource, /* optional */ tagName) {
-      var icon = imageStreamTagAnnotationFilter(resource, "iconClass", tagName);
-      return (icon) ? icon : "fa fa-cube";
-    };
-  })
   .filter('imageName', function() {
```

Comment:
.filter('imageStreamTagTags' is in common and needs to be removed here in console.